torchdrivesim/traffic_controls.py

```diff
@@ -22,13 +27,31 @@ class BaseTrafficControl:
         mask: BxN boolean tensor indicating whether a given traffic control element is present and not a padding.
     """
     def __init__(self, pos: Tensor, allowed_states: Optional[List[str]] = None,
-                 replay_states: Optional[Tensor] = None, mask: Optional[Tensor] = None):
+                 replay_states: Optional[Tensor] = None, mask: Optional[Tensor] = None, ids: Optional[List] = None,
+                 use_mock_lights: bool = False, preset_states: Optional[Dict[int, List[str]]] = None):
         self.pos = pos
+        self.ids = ids
+        self.use_mock_lights = use_mock_lights
         self.allowed_states = allowed_states if allowed_states is not None else self._default_allowed_states()
         self.replay_states = replay_states if replay_states is not None else self._default_replay_states()
         self.mask = mask if mask is not None else self._default_mask()
+        self.preset_states = preset_states
+        self.virtual_overlap = None
 
+        self.pos[...,2] = 1
```

Comment:
Add a comment here that you fix the thickness of the stoplines (or even better make it an argument with default value 1.0).